prow/resultstore/payload.go

```diff
@@ -242,12 +242,16 @@ func startedProperties(started *metadata.Started) []*resultstore.Property {
 	for _, r := range repos {
 		ps = append(ps, &resultstore.Property{
 			Key:   "Repo",
-			Value: "https://" + r,
+			Value: fixRepo(r),
 		})
 	}
 	return ps
 }
 
+// fixRepo deletes the "-review" segment present in some Gerrit Repos to
+// produce a valid URL. Cf. CreateRefs() in gerrit/adapter/adapter.go.
+var fixRepo = strings.NewReplacer("-review.", ".").Replace
```

Comment:
Nit: I don't understand why we're separating this function out and giving it a name if we're not reusing it anywhere.

Reply:
I had two reasons:
1. To explain the rationale for this outside of the loop above where it would detract from readability.
2. strings.NewReplacer() builds an object once that is used over and over. If done in the function, it would repeat object instantiation.

Reply:
Will refactor to only replace ahead of the first dot in the name.

Reply:
Done.